packages/bemjson-to-decl/index.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 const stringifyObj = require('stringify-object');
-const normalize = require('bem-decl').normalize;
-const BemEntity = require('@bem/entity-name');
+const normalize = require('@bem/sdk.decl').normalize;
+const BemEntity = require('@bem/sdk.entity-name');
```

Comment:
is it still proper name for constant?
btw, it's called `b_` in helpers in the same package and `BemEntityName` bellow (which sounds like the best option to me)

Reply:
It's a constructor so yes.

Reply:
`new b_()` ?